### PR TITLE
Create Alpaca docs

### PR DESCRIPTION
**Pull request by anqaka**

### CODE REVIEW

File: docs/magento-setup-guide.md [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+                    'port' => '',
+                    'user' => '',
+                    'password' => '',
+                    'virtualhost' => '/',
+                    'ssl' => ''
+                ]
+            ]
+        ];
+        ```
+
```
Comment (ofrankowska) [inline docs/magento-setup-guide.md:189]: 6. Set up SSL (optional)
    * Run command bellow to enable secure URLs in both the customer-facing and admin-only areas of         your store:
    ```
    bin/magento setup:store-config:set --use-secure=1 --base-url="https://project-name.test/"
    ```
    Adjust `base-url` value.

    * Run command below for to Valet to serve the site over encrypted TLS using HTTP/2:
    ```
    valet secure
    ```

File: docs/alpaca-step-by-step.md [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+  When using **Valet Plus** run commands in main project directory, with **Warden** remember to launch shell session see [Warden usage and common commands](https://docs.warden.dev/usage.html)
+
+1. Add Snowdog Alpaca Packages: <br />
+    ```
+    composer require snowdog/module-alpaca-packages
+    ```
+    See [content of Alpaca Packages here](https://github.com/SnowdogApps/magento2-alpaca-packages)
+2. Install Alpaca Theme latest version:
+    ```
+    composer require snowdog/theme-frontend-alpaca:2.26.*
```
[the remark this answers is not in the document]
Comment (anqaka) [reply docs/alpaca-step-by-step.md:40]: change to version higher than 2.26, thanks

File: docs/alpaca-step-by-step.md [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+
+5. Upgrade application, DB data and schema (_[learn more here](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-db-upgr.html#instgde-cli-db-upgr)_):
+    ```
+    bin/magento setup:upgrade
+    ```
+6. Go to `vendor/snowdog/frontools` and run commands:
+    ````
+    yarn install
+    yarn setup
+    yarn styles && yarn svg && yarn babel
```
[the remark this answers is not in the document]
Comment (anqaka) [reply docs/alpaca-step-by-step.md:58]: right, added

File: docs/magento-setup-guide.md [hunk 4]
```diff
@@ -0,0 +1,377 @@
...
+                                --admin-user="admin" \
+                                --admin-password="Password123" \
+                                --admin-email="name@example.com" \
+                                --admin-firstname="Your First Name" \
+                                --admin-lastname="Your Last Name"
+    ````
+    Adjust options:
+      * `db` fields - name, password and root
+      * Your project local URL and your admin credentials
+4. Create `env` file:
```
[the remark this answers is not in the document]
Comment (anqaka) [reply docs/magento-setup-guide.md:59]: you're right, removed

File: docs/magento-setup-guide.md (hunk 1 above)
Comment (anqaka) [reply docs/magento-setup-guide.md:189]: added

**Pull request merged**